react/components/BaseProductIdentifier.tsx

```diff
@@ -27,25 +28,33 @@ const messages = defineMessages({
   },
 })
 
+const CSS_HANDLES = [
+  'productIdentifier',
+  'productIdentifierLabel',
+  'productIdentifierSeparator',
+  'productIdentifierValue'
+]
```

Comment:
You should use the previous names for these CSS handles.
This array would be:

```typescript
const CSS_HANDLES = [
    'product-identifier',
    'product-identifier__label',
    'product-identifier__separator',
    'product-identifier__value'
] as const
```

Reply:
But then the classes won't change, the idea for the classes is to not have any kind of special characters, right?